src/interpreter/bytecode-generator.cc

```diff
@@ -2999,6 +2999,9 @@ void BytecodeGenerator::VisitConditional(Conditional* expr) {
   ConditionalControlFlowBuilder conditional_builder(
       builder(), block_coverage_builder_, expr);
 
+  // [Replay Instrumentation] We don't emit breakpoints before the call,
+  // because those should generally be emitted by the nesting statement.
```

Comment:
what's the point of this comment?  if there aren't replay related changes in place then we should leave the code alone.